File: py/1187.py
operation = input()
matrix = []

i = 0
while i < 144:
    matrix.append(float(input()))
    i += 1

result = 0
left = 1
right = 11

while True:
    result += sum(matrix[left : right : 1])
    
    left += 1 + 12
    right += 12 - 1

    if left == 66:
        break

if operation == 'S':
    print(f'{(result):.1f}')

else:
    print(f'{(result / 30):.1f}')

File: cpp/1187.cpp
#include <iostream>
#include <stdio.h>

using namespace std;

double sum(double m[], int s, int e)
{
    double sum = 0;

    for (s; s < e; s++) {
        sum += m[s];
    }

    return sum;
}

int main()
{
    int left = 1, right = 11, i;
    double m[144], result = 0;
    char op;
    cin >> op;

    for (i = 0; i < 144; i++) {
        cin >> m[i];
    }

    while (left < 66) {
        result += sum(m, left, right);
        left += 13;
        right += 11;
    }

    if (op == 'S') {
        printf("%.1f\n", result);
    } else {
        printf("%.1f\n", result / 30);
    }

    return 0;
}